source/_components/binary_sensor.w800rf32.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: "W800rf32 Binary Sensor"
+description: "Instructions on how to integrate W800rf32 binary sensors into Home Assistant."
+date: 2018-10-16 12:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: w800rf32.png
+ha_category: Binary Sensor
+ha_release: 0.79
+ha_iot_class: "Local Push"
+---
+
+The `w800rf32` platform supports X10 RF binary sensors such as Palm Pad 
+remotes, key chain remotes, Hawkeye motion detectors, and many, many other X10 RF devices.
+Some that have specifically been used with this are the KR19A keychain, MS16A motion detector
+and the RSS18 four button wall mount keypad.
+
+# Setting up your devices
+
+Once you have set up your [w800rf32 hub](/components/w800rf32/), add the 
+binary sensors to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+binary_sensor:
+  - platform: w800rf32
+    devices:
+      a1:
+        name: motion_hall
+        off_delay: 5
+        device_class: Motion
+      a2:
+        name: motion_kitchen
+        device_class: Motion
+```
+
+Configuration variables:
+
+- **device_class** (*Optional*): The [type or class of the sensor](/components/binary_sensor/) to set the icon in the frontend.
```

Comment:
Please use the new configuration format, see [here](https://developers.home-assistant.io/docs/en/documentation_create_page.html) for meer info.